test/test_pause.js

```diff
@@ -20,338 +20,379 @@ describe('.pause', () => {
     return client.quit();
   });
 
-  it('should pause a queue until resumed', () => {
```

Comment:
Most of the changes in this file are from organizing the tests into "global" and "local" sub-suites. Otherwise, the existing tests have not been modified. 

I can back that out if it's not desired.